en/docs/GettingStarted/key-concepts.md

```diff
@@ -1,275 +1,25 @@
 # Key Concepts
 
-Let's take a look at some concepts and terminology that you need to know in order to follow the use cases.
+| Concept                            | Description                                                  |
+| ---------------------------------- | ------------------------------------------------------------ |
+| <a name="api"></a>API                     | An API (Application Programming Interface) is the fundamental building block of the WSO2 API Manager. It is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and a well documented approach to accessing the exposed service. |
+|  <a name="api-format"></a> API Format                         | Open API Specification (OAS, a.k.a Swagger) format is the underneath representation of an API in the WSO2 API Manager. A swagger definition can be used to import an API to the Publisher Portal. In addition, an API definition in the OAS format can be viewed, edited, imported, or downloaded from the Publisher Portal. |
```

Comment:
Change "the WSO2 API Manager" to "WSO2 API Manager". 

We should not use "the" in front of a product name unless we are referring to some aspect of the product, such as the WSO2 API Manager Management Console.